dcicutils/misc_utils.py

```diff
@@ -11,6 +13,54 @@
 PRINT = print
 
 
+class VirtualApp(TestApp):
```

Comment:
If you're going to inherit from `TestApp`, you don't need to put a TestApp also in the `.virtual_app` slot below. That's an either/or design choice.